supportedBackends/tes/src/main/scala/cromwell/backend/impl/tes/TesAsyncBackendJobExecutionActor.scala

```diff
@@ -148,19 +148,7 @@ class TesAsyncBackendJobExecutionActor(override val standardParams: StandardAsyn
         mode)
     })
 
-    task.map(task => Task(
-      id = None,
-      state = None,
-      name = Option(task.name),
-      description = Option(task.description),
-      inputs = Option(task.inputs),
-      outputs = Option(task.outputs),
-      resources = Option(task.resources),
-      executors = task.executors,
-      volumes = None,
-      tags = None,
-      logs = None
-    ))
+    tesTask.map(TesTask.makeTask)
```

Comment:
Thank you @jgainerdewar for suggesting that I reconsider how to refactor this to make writing the test easier.

Reply:
Nice! I'm glad the test gave us a good excuse to make this change, I like the new way much better.